lib/node_modules/@stdlib/assert/is-same-date-object/benchmark/benchmark.length.js

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
This file needs to be renamed to `benchmark.js`. "Length" benchmark files are reserved for measuring performance on collections.